app/controllers/images_controller.rb

```diff
@@ -26,10 +26,9 @@ def create
         return
       end
 
-      image_revision = ImageUploadService.new(params[:image]).call(current_user)
-
       current_edition = @document.current_edition
       current_revision = current_edition.revision
+      image_revision = ImageUploadService.new(params[:image], current_revision).call(current_user)
```

Comment:
I know this doesn't introduce this here so no biggie or blocker, but I'm pretty confused as to why some args in initialize and others in call. Is there a logic that I'm missing?